File: resolucoes/JavaScript/lista01/ex013.js
const prompt = require('prompt-sync')();

function main() {

const salario = Number(prompt("Digite seu salario:"))

const pos = salario * 0.15

const aumento = salario + pos

console.log(`Seu salario é ${salario} e seu aumento será de ${aumento}`)
}

module.exports = {
    main
}

File: resolucoes/kotlin/lista01/src/main/kotlin/ex013.kt
import java.util.*

internal fun calcularNovoSalario() {
    val scanner = Scanner(System.`in`)
    print("Digite o salário do funcionário: ")
    val salario = scanner.nextDouble()

    // Calcula o valor do aumento
    val aumento = salario * 0.15
    // Calcula o novo salário
    val novoSalario = salario + aumento

    // Exibe o novo salário
    println("O novo salário é: R$${novoSalario}")
}

fun main() {
    calcularNovoSalario()
}
